lms/templates/report_modal.html

```diff
@@ -114,7 +115,7 @@
     $("#report-form").on("ajax:success", function(event, data, status, xhr) {
         $("#report-form-wrapper").css("display", "none");
         $("#report-success-wrapper").css("display", "block");
-        $("#report-success-wrapper").focus();
+        $("#report-success-wrapper .close-modal").focus();
```

Comment:
use this:
<div id="content" role="alert">
...
</div>

$("#content").show();

http://stackoverflow.com/questions/10349987/how-to-notify-screen-readers-using-wai-aria-that-a-div-is-now-visible